public/API/API_GetLeaderboardSubmissions.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ *  API_GetLeaderboardSubmissions - returns submissions for a Leaderboard for the given ID
+ *    i : leaderboardID
+ *    o : offset - number of entries to skip (default: 0)
+ *    c : count - number of entries to return (default: 100, max: 500)
+ *  int         Count                       number of user entries returned in the response
+ *  int         Total                       number of user entries the leaderboard actually has overall
+ *  array       Results
+ *   object      [value]
+ *    int        Rank                       user's leaderboard rank
+ *    string     User                       name of user
+ *    string     Score                      string value of the proper ValueFormat of the leaderboard entry //this feels wrong
+ *    string     DateSubmitted              an ISO8601 timestamp string for when the entry was submitted
```

Comment:
`Y-m-d H:i:s` is not an ISO8601 timestamp.

Reply:
Adjusted to using date('c', $var) instead of date('Y-m-d H:i:s', $var)